OptimizelySDK/Odp/Constants.cs

```diff
@@ -65,7 +65,12 @@ public static class Constants
         /// <summary>
         /// Default message to log when an ODP Event contains invalid data
         /// </summary>
-        public const string ODP_INVALID_DATA_MESSAGE = "ODP data is not valid.";
+        public const string ODP_INVALID_DATA_MESSAGE = "ODP event send failed (event data is not valid).";
```

Comment:
The TDD has a defined message 
odpInvalidData ERROR "ODP data is not valid."

I think we should adhere unless @jaeopt has other input.

Reply:
event data is separate and identifier is is not part of event data.